### PR TITLE
Issue #95: Update `add` command in User Guide

### LINKED ISSUE #95: Update `add` command in UG/DG
To update the `add` command in UG/DG from `v1.2b`s work.

### PR DESCRIPTION
**Pull request by domlimm**
This PR consists of the following changes:
- Update add command usage
- Remove tags related notes

Closes #95.

### CODE REVIEW

**Review by charmainehly — CHANGES_REQUESTED** [review]
Just a quick point on the format of `studentId`!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Command Line Interface (CLI)** while still having the benefits of a Graphical Us
 
    * **`list`** : Lists all TAs.
 
-   * **`add id/E0123456 n/John Doe p/87654321 c/Computer Science`** Adds a new TA into the system.
+   * **`add id/E0123456 n/John Doe p/87654321 c/Computer Science yr/2 avail/1,2,3`** Adds a new TA into the system.
 
    * **`delete A0123456H`** : Deletes the TA with ID A0213456H from the system.
```
[the remark this answers is not in the document]
Comment (domlimm) [reply docs/UserGuide.md:34]: Hey, I think it has yet to be updated.

[the remark this answers is not in the document]
Comment (lzan98) [reply docs/UserGuide.md:34]: @domlimm oops I think that was my part from weeks ago, I'll change it in another PR! Sorry I didn't approve at first cause I thought you were making a change until I realised it was about `delete` (which is my part 😅)

**Review by lzan98 — APPROVED** [review]
LGTM! Good reminder for me to update my sections in the UG too 😆

**Review by charmainehly — APPROVED** [review]
Approved!! LGTM! We'll KIV the format of `studentId`!

**Pull request merged**